packages/payment/integration-tests/__tests__/services/payment-module/index.spec.ts

```diff
@@ -472,25 +472,7 @@ moduleIntegrationTestRunner({
                   data: {},
                   status: "authorized",
                   authorized_at: expect.any(Date),
-                  payment_collection: expect.objectContaining({
-                    id: expect.any(String),
-                  }),
-                  payment: expect.objectContaining({
-                    cart_id: null,
-                    order_id: null,
-                    order_edit_id: null,
-                    customer_id: null,
-                    data: {},
-                    deleted_at: null,
-                    captured_at: null,
-                    canceled_at: null,
-                    refunds: [],
-                    captures: [],
-                    amount: 100,
-                    currency_code: "usd",
-                    provider_id: "pp_system_default",
-                  }),
-                },
+                }),
```

Comment:
q: what is this about?

Reply:
All these other files are fixes because of changing the mikro-orm strategy to JOINED. That should be another PR and I'm to blame for it.